src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_help.py

```diff
@@ -261,3 +261,139 @@
+        - name: --reset
+          short-summary: Remove all the existing conditions before add new conditions. When --reset is given, the values
+                         of --remove-condition will be ignored.
+"""
+
+helps['monitor activity-log alert update-action'] = """
+    type: command
+    short-summary: Update the action groups of this activity log alert.
+    parameters:
+        - name: --name -n
+          short-summary: Name of the activity log alerts
+        - name: --add-action -a
+          short-summary: Add action group. Accept space separated resource id of the action groups to be added. The
+                         invalid resource id is ignored.
+        - name: --remove-action -r
+          short-summary: Remove action group. Accept space separated resource id of the action groups to be removed. The
+                         removing is executed before adding. The invalid resource id is ignored.
+        - name: --reset
+          short-summary: Remove all the existing action groups before add new conditions. When --reset is given, the
+                         values of --remove-action will be ignored.
+        - name: --webhook-properties -w
+          short-summary: Space separated web hook properties in 'key[=value]' format. These properties will be
+                         associated with the action groups added in this command. For any webhook receiver in these
+                         action group, these data are appended to the webhook payload. To attach different webhook
+                         properties to different action groups, add the action groups in separate update-action
+                         commands.
+        - name: --strict
+          short-summary: Fails the command if a action group to be added already exists in the activity log alert and
+                         the given webhook properties are different. If --strict is not set, the webhook properties will
+                         be overridden when the webhook properties are different or erased when webhook properties are
+                         not given.
+"""
+
+helps['monitor activity-log alert update-scope'] = """
+    type: command
+    short-summary: Update the scopes of this activity log alert.
+    parameters:
+        - name: --name -n
+          short-summary: Name of the activity log alerts
+        - name: --add-scope -a
+          short-summary: A list of resourceIds that will be used as prefixes. The alert will only apply to activityLogs
+                         with resourceIds that fall under one of these prefixes.
+        - name: --remove-scope -r
+          short-summary: A list of the scopes to be removed.
+        - name: --reset
+          short-summary: Remove all the existing scopes before add new scopes. When --reset is given, the values of
+                         --remove-scope will be ignored.
+"""
+
+helps['monitor activity-log alert update'] = """
```

Comment:
Most of the commands in this group should be folded into this command. This command should also be a generic update.